packages/angular/plugins/component-testing.ts

```diff
@@ -322,6 +322,7 @@ function withSchemaDefaults(options: any): BrowserBuilderSchema {
   options.outputHashing ??= 'none';
   options.progress ??= true;
   options.scripts ??= [];
+  options.main = options.main ??= options.browser;
```

Comment:
```js
  options.main ??= options.browser;
```

Reply:
Good catch, removed the check here: #20382

I like the co-location of the checks that throw errors more then having only a single condition/branch.